src/diffusers/models/lora.py

```diff
@@ -57,25 +66,6 @@ def text_encoder_mlp_modules(text_encoder):
     return mlp_modules
 
 
-def text_encoder_lora_state_dict(text_encoder):
```

Comment:
This is only used in training scripts, let's remove it from here. It'll make it easier to later completely remove once the training scripts are refactored.